src/Common/ZooKeeper/ZooKeeperImpl.cpp

```diff
@@ -1217,7 +1217,7 @@ void ZooKeeper::close()
     RequestInfo request_info;
     request_info.request = std::make_shared<ZooKeeperCloseRequest>(std::move(request));
 
-    if (!requests_queue.tryPush(std::move(request_info), operation_timeout.totalMilliseconds()))
```

Comment:
Better to use my pull request, it seems that you just auto fix issue, I added necessary move constructors by hand.